aws/sdk/examples/dynamodb/src/bin/dynamodb-helloworld.rs

```diff
@@ -7,6 +7,7 @@ use dynamodb::model::{
     AttributeDefinition, KeySchemaElement, KeyType, ProvisionedThroughput, ScalarAttributeType,
 };
 
+/// Lists your DynamoDB tables and creates the table __test_table__.
```

Comment:
I think the table `test-table` not `__test_table__`?

Reply:
It's a doc comment, so I want the table name to be bold in the docs.

Reply:
I think it's still `test-table`. I think `__` is italic, you probably want `**`
`**test-table**` = **test-table**

Reply:
[note that this is still todo]

Reply:
Two underscores also make it bold.

Reply:
the table name is wrong. It says `test_table` but the table is `test-table`

Reply:
Fixed.